components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -214,36 +242,62 @@ class InvitationModalMembersStep extends React.PureComponent {
                         <UsersEmailsInput
                             usersLoader={this.usersLoader}
                             placeholder={placeholder}
-                            ariaLabel={localizeMessage('invitation_modal.members.search_and_add.title', 'Invite People')}
+                            showError={this.shouldShowPickerError()}
+                            ariaLabel={localizeMessage(
+                                'invitation_modal.members.search_and_add.title',
+                                'Invite People',
+                            )}
                             onChange={this.onChange}
                             value={this.state.usersAndEmails}
-                            validAddressMessageId={t('invitation_modal.members.users_emails_input.valid_email')}
+                            validAddressMessageId={t(
+                                'invitation_modal.members.users_emails_input.valid_email',
+                            )}
                             validAddressMessageDefault='Invite **{email}** as a team member'
+                            errorMessageId={
+                                'invitation_modal.invite_members.hit_cloud_user_limit'
```

Comment:
You'll want to add `t` to this id as well to make sure our marker function picks it up as a translation.